### ISSUE TITLE
Broken links in invoices documentation

### ISSUE THREAD

**Issue by ila** [pgeu/pgeu-system#64]
This [file](https://github.com/pgeu/pgeu-system/blob/master/docs/invoices/payment.md) contains broken links. Specifically, I am talking about the first two hyperlinks.

**Comment by mhagander** [comment]
This seems to be because the links are set up for being used in the help viewer of the system. They work there, but they will fail when you read the documentation on github, which I'm assuming is what you're doing.

It would be good if we can figure out a way to make it work in both. This is probably doable by rewriting the links when rendered on the website.

The view is in https://github.com/pgeu/pgeu-system/blob/master/postgresqleu/util/docsviews.py. We're already hacking up both SVG images and the title of the page using regexps there, so it seems reasonable we should be able to put something together that allowed us to use "github compatible" links in the file itself and then rewrite them for showing on the site.

It's unfortunately a bit more involved than just fixing a broken links, so it's not a 30 second fix.

**Comment by Saurav-Shrivastav** [comment]
@mhagander The first link (for `accounting`), doesn't work on the local system too. The url - `http://127.0.0.1:8000/admin/docs/accounting/` returns a page 404 not found error. Though the second link works fine at `http://127.0.0.1:8000/admin/docs/jobs`. So, which page is `accounting` actually referring to?

**Comment by sfrost** [comment]
Yeah, I can see on at least one of our test instances that the first link doesn't actually work, so it isn't just a github rendering issue.  Not sure what it should be referring to though...

**Comment by mhagander** [comment]
I think the reason the accounting link doesn't work is very simple -- that documentation is simply not written yet :) We should perhaps just stick an empty placeholder in there to fix the link itself, basically documenting the lack of documentation.
